sections/tox.zsh

```diff
@@ -0,0 +1,54 @@
+#
+# Tox
+#
+# tox is a generic virtualenv management and test command line tool
+# Link: https://tox.readthedocs.io/
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_TOX_SHOW="${SPACESHIP_TOX_SHOW=true}"
+SPACESHIP_TOX_PREFIX="${SPACESHIP_TOX_PREFIX=""}"
+SPACESHIP_TOX_SUFFIX="${SPACESHIP_TOX_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
+SPACESHIP_TOX_SYMBOL="${SPACESHIP_TOX_SYMBOL="  "}"
```

Comment:
Beware that upstream will not accept Font Awesome icons as default values, better have some standard Unicode icon + one space, and override locally.

Reply:
Ok